src/sections/Footer/footer.style.js

```diff
@@ -1,199 +1,236 @@
-          line-height: 60px;
-          border-radius: 8px;
-          border: 1px solid ${props => props.theme.white};
-          text-align: center;
-          transition: 450ms all;
-          &:hover {
-            color: ${props => props.theme.white};
-            background: rgb(251, 123, 129);
-            background: -moz-linear-gradient(
-              left,
-              rgba(251, 123, 129, 1) 0%,
-              rgba(254, 191, 179, 1) 100%
-            );
-            background: -webkit-linear-gradient(
-              left,
-              rgba(251, 123, 129, 1) 0%,
-              rgba(254, 191, 179, 1) 100%
-            );
-            background: linear-gradient(
-              to right,
-              rgba(251, 123, 129, 1) 0%,
-              rgba(254, 191, 179, 1) 100%
-            );
-            filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#fb7b81', endColorstr='#febfb3',GradientType=1 );
-            border-color: #febfb3;
-          }
-        }
-      }
-      li + li {
-        margin-left: 30px;
-      }
-    }
-    .widget-catagory {
-      li {
-        .anchor {
-          font-size: 18px;
-          line-height: 22px;
-        }
-      }
-      li + li {
-        margin-top: 15px;
-      }
-    }
-  }
-  .footer-bottom{
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    padding-top: 50px;
-    margin-top: 100px;
-    border-top: 1px solid rgba(255, 255, 255, 0.5);
-    .footer-menu{
-      order: 2;
-      li {
-        display: inline-block;
-        font-size: 18px;
-      }
-      li + li{
-        margin-left: 40px;
-      }
-    }
-    p{
-      order: 1;
-      margin: 0;
-    }
-  }
-  @media only screen and (max-width: 1600px) {
-    .section__particle{
-      &.one{
-          display: none;
-      }
-    } 
-  }
-  @media only screen and (max-width: 1280px) { 
-    .footer-widgets {
-      .social {
-        li + li {
-          margin-left: 15px;
-        }
-      }
-    }
-  }
-  @media only screen and (max-width: 912px) {
-    background-position: -200px; 
-    .footer-widgets {
-      margin-bottom: 70px;
-    }
-    .footer-bottom {
-      display: block;
-      margin-top: 30px;
-     .footer-menu {
-        margin-bottom: 40px;
-      }
-      .footer-menu,
-      .copyright-text {
-        text-align: center;
-      }
-    }
-  }
-  @media only screen and (max-width: 568px) {
-    background-position: 10%;
-    text-align: center;
-    .footer-widgets {
-      .info {
-        li {
-          justify-content: center;
-        }
-      }
-    }
-  }
-  @media only screen and (max-width: 375px) {
-    .footer-bottom {
-      .footer-menu {
-        li {
-          display: block;
-        }
-        li + li {
-          margin-left: 0;
-          margin-top: 30px;
-        }
-      }
-    }
-  }
-`;
-
-export default FotterWrapper;
+import styled from 'styled-components';
+import FooterBG from '../../assets/images/app/footer/footer-bg.png';

```

Comment:
Can we reduce the size of this background image to 1 pixel and have it repeated horizontally and vertically instead, so that we can achieve faster load time and higher page rank? Even better, can we just use CSS to color the background?

More so than either of these two comments, can we get back to something similar to the original theme? If not the original theme, then something with more activity (more curves and gradients)?

<img width="1704" alt="Screen Shot 2020-08-02 at 10 03 02 PM" src="https://user-images.githubusercontent.com/7570704/89142048-f6313d00-d50b-11ea-9425-e13e7d7008f6.png">

<img width="1701" alt="Screen Shot 2020-08-02 at 10 00 53 PM" src="https://user-images.githubusercontent.com/7570704/89142056-ffbaa500-d50b-11ea-9bcd-91742911debf.png">